docs/appendix/common-problems.md

```diff
@@ -23,6 +23,22 @@ For Mac users, the easiest way is to open a new terminal and run the following c
     rm -rf ~/Library/caveData
 
 
+---
+
+## Disappearing Configurations
+
+If you ever notice some of the following settings you've configured/saved disappear from CAVE:
+
+- Saved Displays or Procedures
+- NSHARP settings (line thickness, etc)
+- Colormap settings
+- StyleRule settings
+
+!!! note "This is not a fully exhaustive list, so if something else has dissappeared it might be the same underlying issue still."
+
+Then it is likely we have recently changed our production EDEX server.
```

Comment:
do we need to go into any more detail, do people know/care what we mean by "changing our production EDEX server"

Reply:
I think this should be fine.